build-mbl/build.sh

```diff
@@ -705,6 +712,10 @@ if [ "${flag_binary_release}" -eq 1 ]; then
     fi
 fi
 
+if [ -n "${mcc_destdir:-}" ] && [ -z "${inject_mcc_files:-}" ]; then
+  printf "error: --mcc-destdir requires at least one --inject-mcc parameter.\n" >&2
```

Comment:
Let's process mcc_destdir here and decide here which is the base path where to copy files. The inject should be the same.

Reply:
See what do you think: 87997ce